test/clinical-clusters/clinical-clusters-spec.js

```diff
@@ -36,4 +36,74 @@ describe('benchmarks getter functions', () => {
     var cluster052 = clusters[0];
     assert.deepEqual(['130'], cluster052.clinicalClusters[0].measureIds);
   });
+
+  it('claims 117 should not have any clinical clusters', () => {
+    var data = main.getClinicalClusterData();
+    var clusters = data.filter(c => c.measureId === '117' && c.submissionMethod === 'claims');
+    assert.equal(0, clusters.length);
```

Comment:
I see it in the docs but looks like the chai version we may be using is not compatible.
```
TypeError: assert.isEmpty is not a function
    at Context.it (test/clinical-clusters/clinical-clusters-spec.js:43:12)
```